app/views/projects/new.html.slim

```diff
@@ -12,7 +11,7 @@ p.well Please make sure you have read our #{link_to "mentor guide", 'http://rail
       li.active
         a (href="#project_pane" data-toggle="tab") Project
       li
-        a (href="#preview_pane" data-toggle="tab" id="preview_tab" onclick="sendPreview()") Preview
+        a (href="#preview_pane" data-toggle="tab" id="preview_tab" onclick="sendPreview()" data-ajax_path=preview_projects_path) Preview
```

Comment:
Just one minor thing: can you use a dash instead of an underscore here? And, concordantly, in the corresponding coffee script as well?